linux/shim/io.go

```diff
@@ -15,8 +15,9 @@ import (
 )
 
 func copyConsole(ctx context.Context, console console.Console, stdin, stdout, stderr string, wg, cwg *sync.WaitGroup) error {
+	fctx := context.Background()
 	if stdin != "" {
-		in, err := fifo.OpenFifo(ctx, stdin, syscall.O_RDONLY, 0)
+		in, err := fifo.OpenFifo(fctx, stdin, syscall.O_RDONLY, 0)
```

Comment:
I also just realized. We actually need a way to honor the context passed down from the caller.

Those open are blocking (i.e. `O_NONBLOCK` is not set on purpose), meaning that if the context of the RPC get canceled, we will get stuck here forever